../lowlevel/gen_bb_mul_code.c: In function 'main':
../lowlevel/gen_bb_mul_code.c:95:17: error: C++ style comments are not allowed in ISO C90
   95 |                 // continue;    /* mul_1_n is no longer used */
      |                 ^

--- a/lowlevel/gen_bb_mul_code.c
+++ b/lowlevel/gen_bb_mul_code.c
@@ -92,7 +92,6 @@
 	    printf("GF2X_STORAGE_CLASS_mul1 void\n");
 	    printf("gf2x_mul1 (unsigned long *c, unsigned long a, unsigned long b)\n");
 	} else if (fn == 1) {
-		// continue;	/* mul_1_n is no longer used */
 	    printf("GF2X_STORAGE_CLASS_mul_1_n unsigned long\n");
 	    printf
 		("gf2x_mul_1_n (unsigned long *cp, const unsigned long *bp, long sb, unsigned long a)\n");
